counterexamples/buildings/bad-floors-underground.yaml

```diff
@@ -0,0 +1,11 @@
+---
+id: overture:buildings:footprint:1234
+type: Feature
+geometry:
+  type: Polygon
+  coordinates: [[[0, 0], [0, 1], [1, 1], [1, 0], [0, 0]]]
+properties:
+  theme: buildings
+  type: building
```

Comment:
- Missing `update_time` which means it's schema-invalid even without the `num_floors_underground`.
- Would you mind adding the `ext_expected_errors: []` property?
    - That property acts like an assertion to verify that the schema validator fails with the error you are expecting.
    - It's insurance against counterexamples that are invalid, but for the wrong reason. There should be lots of examples of use in other nearby counterexample files.

Reply:
Still seems to be missing `update_time` and `ext_expected_errors`...

Reply:
I added `update_time` — However, where can I find what the expected errors should look like? Because mine look different...